docs/en_US/Compressor/Pruner.md

```diff
@@ -92,3 +92,55 @@ You can view example for more information
 
 ***
 
+## Filter Pruner
+
+In ['PRUNING FILTERS FOR EFFICIENT CONVNETS'](https://arxiv.org/abs/1608.08710), authors Hao Li, Asim Kadav, Igor Durdanovic, Hanan Samet and Hans Peter Graf.
```

Comment:
this is a one-shot pruning or iterative pruning?

Reply:
one-shot pruning

Reply:
I mean it would be better to make it clear in doc...